src/components/views/dialogs/CreateRoomDialog.tsx

```diff
@@ -349,6 +358,7 @@ export default class CreateRoomDialog extends React.Component<IProps, IState> {
                         <JoinRuleDropdown
                             label={_t("Room visibility")}
                             labelInvite={_t("Private room (invite only)")}
+                            labelKnock={this.knockingEnabled ? _t("Anyone can knock to join") : undefined}
```

Comment:
Designs seem to suggest a different label - see https://www.figma.com/file/7TqjqdMBaPpm3IavKsMYu6/%5BBWI%5D-Knocking?node-id=367%3A57764